Dockerfile

```diff
@@ -1,12 +1,12 @@
 # syntax=docker/dockerfile:1.3
-FROM ocaml/opam:alpine-3.15-ocaml-4.14 as build-opam2web
+FROM ocaml/opam:alpine-3.20-ocaml-4.14 as build-opam2web
 RUN sudo apk add g++ gmp-dev
-COPY . /home/opam/opam2web
+COPY --chown=opam:opam . /home/opam/opam2web
 WORKDIR /home/opam/opam2web
 ENV OCAMLRUNPARAM b
 RUN sudo mkdir -p /opt/opam2web && sudo chown opam:opam /opt/opam2web
+RUN sudo mv /usr/bin/opam-2.2 /usr/bin/opam && opam update
 RUN opam repo set-url default git+https://github.com/ocaml/opam-repository.git#${OPAM_REPO_GIT_SHA}
-ENV OPAMSOLVERTIMEOUT 120
```

Comment:
I would advise to keep this just in case the machine it's building with is too busy with other tasks

Reply:
ok, I've reinstated the 120 second timeout.